app.py

```diff
@@ -345,7 +345,23 @@ def snap_release(snap_name):
     return publisher_views.snap_release(snap_name)
 
 
-@app.route('/account/snaps/<snap_name>/market', methods=['POST'])
+@app.route('/account/snaps/<snap_name>/listing', methods=['POST'])
+@login_required
+def post_listing_snap(snap_name):
+    return publisher_views.post_listing_snap(snap_name)
+
+
+@app.route('/account/snaps/<snap_name>/market')
+@login_required
+def get_market_snap(snap_name):
+    return flask.redirect(
+        "/account/snaps/{snap_name}/listing".format(
+            snap_name=snap_name))
+
+
+@app.route('/account/snaps/<snap_name>/measure')
 @login_required
-def post_market_snap(snap_name):
-    return publisher_views.post_market_snap(snap_name)
+def get_measure_snap(snap_name):
```

Comment:
Shouldn't it be `get_metrics_snap` `get_listing_snap` - or maybe even `get_snap_metrics` `get _snap_listing`?

Reply:
that's just the redirect to metrics, figured would leave them as the legacy name and keep the name formats the same throughout